packages/helix-shared-tokencache/test/getCachePlugin.test.js

```diff
@@ -117,4 +117,50 @@ describe('getCachePlugin tests', () => {
     assert.ok(cachePlugin);
     assert.strictEqual(cachePlugin.location, 'helix-content-bus/default/.helix-auth/auth-onedrive-content.json');
   });
+
+  it('does not store back unmodified auth settings', async () => {
```

Comment:
new test that simulates what a long running service does: creates a plugin, reads from S3, and re-creates another plugin, reading back from shared global cache